lib/droplet_kit/resources/kubernetes_resource.rb

```diff
@@ -24,26 +24,39 @@ class KubernetesResource < ResourceKit::Resource
       action :delete, 'DELETE /v2/kubernetes/clusters/:cluster_id' do
       end
 
-      action :cluster_node_pools, 'GET /v2/kubernetes/clusters/:cluster_id/node_pools' do
+      action :cluster_node_pools, 'GET /v2/kubernetes/clusters/:id/node_pools' do
```

Comment:
Would it be possible to drop the word `cluster` from here, I think we'll want the base resource to be `KubernetesCluster` instead of `Kubernetes` (I'll comment that on that in the other PR), so then this would become redundant.

In other words, I think this should be:
```
action :node_pools, ...
```